service-workers/service-worker/active.https.html

```diff
@@ -5,51 +5,55 @@
 <script src="resources/test-helpers.sub.js"></script>
 <body>
 <script>
+
+const SCRIPT = 'resources/empty-worker.js';
+const SCOPE = 'resources/blank.html';
+let registration;
+
 // "active" is set
-async_test(function(t) {
-    var step = t.step_func.bind(t);
-    var url = 'resources/empty-worker.js';
-    var scope = 'resources/blank.html';
-    var frame;
-    var registration;
+promise_test(async t => {
+    let step = t.step_func.bind(t);
+    let frame;
+
+    await service_worker_unregister(t, SCOPE);
+    frame = await with_iframe(SCOPE);
+    registration =
+        await navigator.serviceWorker.register(SCRIPT, {scope: SCOPE});
```

Comment:
how about removing lines 11 and 16 and saying const frame = and const registration = here?